### DISCUSSION TITLE
Welcome to RubixML Discussions!

### DISCUSSION THREAD

**Discussion by andrewdalpino** [RubixML/ML#138] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by auth1299** [comment]
I'm trying to use rubix ML for PHP but having issues. my dataset is all strings. I'm looking to find clusters (kmeans). How do I pre-process the dataset so I can find clusters in my data?  From how I understand it, kmeans requires integer values, correct?

**Comment by diegoxa** [comment]
hi @andrewdalpino, subdomain https://docs.rubixml.com/ is redirecting to a random site (Malvertising)

<img width="1626" height="451" alt="image" src="https://github.com/user-attachments/assets/b4d69eb2-578e-4aa0-8217-6cb3060a173b" />
